appveyor.yml

```diff
@@ -26,7 +26,7 @@ cache:
   - dhall\.stack-work -> '%STACK_YAML%'
   - dhall-json\.stack-work -> '%STACK_YAML%'
   - dhall-text\.stack-work -> '%STACK_YAML%'
-  - dhall-bash\.stack-work -> '%STACK_YAML%'
+  - dhall-lsp-server\.stack-work -> '%STACK_YAML%'
```

Comment:
Did you mean to remove the line for `dhall-bash`?

Reply:
I removed it accidentally, thanks for note it